frontend_tests/casper_tests/10-admin.js

```diff
@@ -185,6 +185,181 @@ casper.then(function () {
 });
 // TODO: Test stream deletion
 
+// Test turning message editing off and on
+function then_edit_last_message() {
+    casper.then(function () {
+        casper.evaluate(function () {
+            var msg = $('#zhome .message_row:last');
+            msg.find('.info').click();
+            $('.popover_edit_message').click();
+        });
+    });
+    casper.waitForSelector(".message_edit_content");
+}
```

Comment:
You should move this to `frontend_tests/casper_lib/common.js` rather than duplicating the version in `08-edit.js`.